### ISSUE TITLE
ChangeLog or NEWS file

### ISSUE THREAD

**Issue by hpdeifel** [minad/corfu#33]
Hi,

I like the approach of `corfu` and consider replacing `company` with it, but it still lacks (or lacked) some features. So I'd like to check in from time to time to see what's new. Unfortunately, currently the commit log is quite busy and hard to parse for me. A NEWS file (or ChangeLog, or annotated git-tags, or whatever) would be nice for this purpose.

Of course, this is suggesting additional thankless work for the maintainer, so please take it as a humble wish and feel free to ignore it.

**Comment by minad** [comment]
> I like the approach of corfu and consider replacing company with it, but it still lacks (or lacked) some features.

Which features do you miss?

I will probably add a changelog at some point, for now you can look at the [Features list](https://github.com/minad/corfu#features) in the README, which is up-to-date.

**Comment by hpdeifel** [comment]
I think at the time it was mostly automatic completion and some problematic interaction with lsp-mode. I haven't tried it since.

Thanks for considering adding a changelog and for the pointer to the features list.

**Comment by alphapapa** [comment]
@minad Please consider adding a changelog file, or a heading to the readme.  Without one, users can't know what new features to look for when upgrading.  For example, I just upgraded from 0.26 to 0.28, and I've no idea what's changed, and in order to find out, I'd have to dig through commit logs to sift the wheat from the chaff.  Not having one makes the package much less useful to users who continue to use the package as it matures.
